rx_paparazzo/src/main/java/com/fuck_boilerplate/rx_paparazzo/RxPaparazzo.java

```diff
@@ -107,6 +108,14 @@ public BuilderImage(T ui) {
         }
 
         /**
+         * Call it when crop features is required.
+         */
+        public BuilderImage<T> cropFeatures(UCropFeature feature) {
```

Comment:
This method should be named `crop`, being that way an overloaded version of existing `crop`.